reference/authorization.yaml

```diff
@@ -88,7 +88,7 @@ paths:
         ### Get a refresh token using code
 
         This option is only useful for OAuth 2.0 apps. After the user authorizes the app, your backend would've received a `code`. Use this API
-        with the `grant_type` set to `code` and the `code` parameter with the code value to get a refresh token.
+        with the `grant_type` set to `authorization_code` and the `code` parameter with the code value to get a refresh token.
```

Comment:
can we also add a line for token refresh?
Something like 
If you already have a refresh token set `grant_type` as `refresh_token` to get the latest access token

Reply:
okies